src/App.js

```diff
@@ -9,7 +10,13 @@ const article = {
 };
 
 const App = () => (
-  <h1>React Article</h1>
+  <main>
+    <Article
+      title={article.title}
+      text={article.text}
+      date={article.date}
+    />
```

Comment:
or you can do it like this
https://medium.com/@kevinyckim33/jsx-spread-operator-component-props-meaning-3c9bcadd2493

```suggestion
    <Article {...article} />
```